flocker/ca/functional/test_validation.py

```diff
@@ -241,6 +246,40 @@ class ControlServicePolicyValidationTests(make_validation_tests(
     """
```

Comment:
This original TestCase seems identical to the new `ControlServicePolicyIPValidationTests` case below. Just rename it instead?